src/js/views/clinicians/clinicians-all_views.js

```diff
@@ -149,13 +168,56 @@ const ListView = CollectionView.extend({
   className: 'table-list',
   tagName: 'table',
   childView: ItemView,
-  emptyView: EmptyView,
+  emptyView() {
+    if (this.state.get('searchQuery')) {
+      return EmptyFindInListView;
+    }
+
+    return EmptyView;
+  },
   collectionEvents: {
     'change:name': 'sort',
   },
+  childViewTriggers: {
+    'render': 'listItem:render',
+  },
   viewComparator({ model }) {
     return String(model.get('name')).toLowerCase();
   },
+  initialize({ state }) {
+    this.state = state;
+
+    this.listenTo(state, 'change:searchQuery', this.searchList);
+  },
+  onListItemRender(view) {
+    view.searchString = view.$el.text();
+  },
+  onRenderChildren() {
+    this.triggerMethod('filtered', this.children.pluck('model'));
+  },
+  searchList(state, searchQuery) {
+    if (!searchQuery) {
+      this.removeFilter();
+      return;
+    }
+
+    const matchers = this._buildMatchers(searchQuery);
+
+    this.setFilter(function({ searchString }) {
+      return every(matchers, function(matcher) {
+        return matcher.test(searchString);
+      });
+    });
+  },
+  _buildMatchers(searchQuery) {
```

Comment:
Possibly move this into a utility function that is imported and used?

This same function is also used in the `src/js/views/patients/schedule/schedule_views.js` and `src/js/views/patients/worklist/worklist_views.js` files.

Reply:
Yeah I think that's a great idea.  Let's do it.

Reply:
Per our discussion on Slack, we'll leave this as-is for this pull request. And we'll address this change in a separate Shortcut story.